pkgs/development/python-modules/chevron/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "chevron";
+  version = "0.13.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "f95054a8b303268ebf3efd6bdfc8c1b428d3fc92327913b4e236d062ec61c989";
+  };
+
+    meta = with lib; {
+    homepage = https://github.com/awslabs/aws-serverlessrepo-python;
+    description = "A python implementation of the mustache templating language.";
+    license = lib.licenses.mit;
```

Comment:
```suggestion
    license = licenses.mit;
```

Reply:
Fixed in cd688755f84ee4c9da816d2144a7d1eeb901086f.